core/mondoo-windows-inventory.mql.yaml

```diff
@@ -137,3 +137,9 @@ packs:
         title: Logged-in users
         mql: |
           parse.json(content: powershell("Get-Process -IncludeUserName explorer | Select-Object Username | ConvertTo-Json").stdout).params
+      - uid: mondoo-windows-exchange-server-version
+        title: Exchange Server Version
+        filters: |
+          package('Microsoft Exchange Server').installed
+        mql: |
+         powershell('(Get-Command ExSetup.exe | ForEach-Object { $_.FileVersionInfo } | Select-Object -First 1).FileVersion').stdout
```

Comment:
Is this not something we can get from the package data?

Reply:
@tas50 yes but then we have to provide the customer a list of packages.

this case is easy

```
packages.where(name == /Exchange Server.*Cumulative Update.*/ || name == 'Microsoft Exchange Server') { name version }
packages.where.list: [
  0: {
    version: "15.1.2507.6"
    name: "Microsoft Exchange Server 2016 Cumulative Update 23"
  }
  1: {
    version: "15.1.2507.6"
    name: "Microsoft Exchange Server"
  }
]
```

but there exists cases where the version of the Exchange server differs from the Version of the Cumulative Update, like this case:

```
packages.where(name == /Exchange Server.*Cumulative Update.*/ || name == 'Microsoft Exchange Server') { name version }
packages.where.list: [
  0: {
    version: "15.2.1544.9",
    name: "Microsoft Exchange Server 2019 Cumulative Update 14",
  }
  1: {
    version: "15.2.1544.4"
    name: "Microsoft Exchange Server"
  }
]
```

@tas50 do you prefer this query: `packages.where(name == /Exchange Server.*Cumulative Update.*/ || name == 'Microsoft Exchange Server') { name version }`

Reply:
If we can't get the data correctly with package then this makes sense. Just double checking